### DISCUSSION TITLE
Setting up IntelliJ IDEA with DDEV

### DISCUSSION THREAD

**Discussion by madamadami** [ddev/ddev#6530] [Q&A] [answered]
For the first time, I want to set up IntelliJ IDEA with DDEV. Since phpStorm and IntelliJ IDEA are very similar, I followed this doc https://ddev.readthedocs.io/en/stable/users/install/phpstorm/
But when specifying docker compose executable, I get this error: "Docker Compose V1 is deprecated, please use Docker
Compose V2." This is cumbersome, because I have version 2.29.1 already installed. What is the reason for this error and how I can fix it?

My system:
macOS 14.6.1
docker provider: orbstack 1.7.2 (17389)

![version](https://github.com/user-attachments/assets/44e0417f-7dc5-460f-9b72-b7945b287b4f)

**Comment by rfay** [comment, marked as answer]
Hi @madamadami - Jetbrains is still a bit confused about what docker-compose v2 is. You have the right path there it looks like to me. 

DDEV does not use any Docker Compose that you may have installed. It uses its own `docker-compose`, which you have set up correctly there. 

I opened an issue with them about this. But they think that using `docker-compose` means using compose v1, and they think that using `docker compose` means compose v2, but it's just something they've been confused about for a few years.

https://youtrack.jetbrains.com/issue/IJPL-68723/Misundertanding-Miscommunication-of-docker-compose-v2

I should also mention that if you use the DDEV Integration Plugin, it does all these things for you, https://plugins.jetbrains.com/plugin/18813-ddev-integration

**Reply by madamadami** [reply]
Thanks a lot! Finally I used the integration plugin.

**Reply by rfay** [reply]
Yay!
